website/src/components/SearchPage/DownloadDialog/DownloadButton.tsx

```diff
@@ -90,7 +101,7 @@ export const DownloadButton: FC<DownloadButtonProps> = ({
     return (
         <div className='flex items-center'>
             <a
-                className={`btn loculusColor ${disabled ? 'btn-disabled' : ''} text-white`}
+                className={`btn loculusColor ${disabled || handleClick === undefined ? 'btn-disabled' : ''} text-white`}
```

Comment:
I think we probably need a message here ideally saying "Sorry, we don't yet support displayName downloads for long queries" or something

Reply:
Sounds good. I'll do that in a separate PR.